tests/specs/concrete-rules.txt

```diff
@@ -5,8 +5,6 @@ EVM.Cgascap
 EVM.Cmem
 EVM.Csstore.new
 EVM.Csstore.old
-EVM.#lookup.none
```

Comment:
This is probably related to my PR. Thanks for pointing out: https://github.com/kframework/evm-semantics/pull/852 If they don't cause any test failures let me fix this part pls.

Reply:
@denis-bogdanas I made this as a separate PR: https://github.com/kframework/evm-semantics/pull/861